src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -145,10 +157,20 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
       return;
     }
     this.entryForm.patchValue({ start_date: newHourEntered });
-    this.store.dispatch(new entryActions.UpdateCurrentOrLastEntry({ ...this.newData, ...this.entryForm.value }));
+    if (this.isTestUser) {
+      this.newData.owner_id = this.getOwnerId();
+      this.newData.update_last_entry_if_overlap = true;
```

Comment:
I think this variable should not be here. There is an action called, update entry running. Try putting this assigment there.